docs/user/rstudio/ide/guide/publish/connecting.qmd

```diff
@@ -28,21 +34,24 @@ install.packages("rsconnect")
 
 ![Global options menu](images/connecting-1.png){fig-alt="Global options menu"}
 
-You'll need to connect RStudio to the remote hosting environment (e.g. Posit Connect, Shinyapps.io, RPubs.com) and authorize RStudio to use your account. To add a publishing account to RStudio, press **Tools** in the main menu, then **Global Options\...**.
+You'll need to connect RStudio to the remote hosting environment (e.g. Posit Connect, Shinyapps.io, Posit Cloud, RPubs.com) and authorize RStudio to use your account. To add a publishing account to RStudio, press **Tools** in the main menu, then **Global Options\...**.
 
 In the Options panel that appears, choose **Publishing** in the sidebar, and use the **Connect\...** button to add a new account.
 
 ![](images/connecting-publishing.png){fig-alt="The Global options menu, highlighting the Publishing tab and the Connect... button"}
 
-
 Select the publishing destination, then follow the **Connect Account** wizard's instructions for that particular product.
 
 ![Publishing account configuration](images/connect-account.png){fig-alt="Publishing account configuration"}
 
 Log in and click **Connect** to authorize the IDE to deploy on your behalf. Then return to the IDE and click the **Connect Account** button.
 
-Once you have successfully configured the connection within RStudio, you may now publish content to Posit Connect or Shinyapps.io.
+Once you have successfully configured the connection within RStudio, you may now publish content to Posit Connect, Posit Cloud, RPubs, or Shinyapps.io.
+
+Full Documentation for:
```

Comment:
I'm not sure why we repeat the full documentation and links here at top and bottom of page, but it's fine if we feel that makes it easier to see for the user

Reply:
Yah, it feels duplicative but I wanted to very specifically guide people to external docs.